globals.d.ts

```diff
@@ -1,9 +1,12 @@
 export {};
 
+import  { IJitsiMeetScreenObtainer } from './modules/RTC/ScreenObtainer'
+
 declare global {
     type Timeout = ReturnType<typeof setTimeout>;
     interface Window {
-                JitsiMeetJS?: {
+        JitsiMeetScreenObtainer?: IJitsiMeetScreenObtainer;
```

Comment:
Drop this, it's for a legacy code that will go away soon. Add a ts-ignore in ScreenObtainer instead.